api/controllers/v1/slack.ts

```diff
@@ -140,9 +140,11 @@ export const getDeployableJobs = async (values, entitlement, branchRepository: B
       urlSlug,
       false,
       false,
-      '-g'
+      false
     );
 
+    newPayload.stable = isStableBranch ? true : false;
```

Comment:
I think the `!!isStableBranch` incantation is idiomatic for this. or `new Boolean(isStableBranch)`.

Reply:
ooh nice. done.